crates/rspack_core/Cargo.toml

```diff
@@ -23,7 +23,7 @@ itertools = { workspace = true }
 mime_guess = { workspace = true }
 nodejs-resolver = { version = "0.1.0" }
 once_cell = { workspace = true }
-oxc_resolver = { version = "0.5.3", features = ["tracing-subscriber"] }
```

Comment:
This will break RSPACK_PROFILE since there are two tracing subscriber in the global, cc @Boshen

Reply:
<img width="1039" alt="Screenshot 2023-11-16 at 17 29 39" src="https://github.com/web-infra-dev/rspack/assets/42857895/7ef3d34c-1d73-49b7-a73d-7a92d93eaf9d">

Reply:
oxc_resolver has `tracing` in it, let's try and enable it on the rspack side.